Simulate updater to fix failing apt tests

deeplow opened the pull request:
The Qubes CI doesn't update qubes right after installing the worsktation. Therefore, when checking if apt targets are correct (via apt-get indextargets) it fails, due to a missing "apt-get update".

Fixes https://github.com/freedomofpress/securedrop-workstation/issues/1333

Commit:
Simulate updater to fix failing apt tests

The Qubes CI doesn't update qubes right after installing the
worsktation. Therefore, when checking if apt targets are correct (via
apt-get indextargets) it fails, due to a missing "apt-get update".

Fixes https://github.com/freedomofpress/securedrop-workstation/issues/1333

eloquence commented:
@mig5 - any thoughts on how to best test this change without merging it? (We're aiming to retire this infra soon, but still relying on it in the meantime, this will hopefully help squash a flaky test.)

deeplow commented:
> >  Once done, you could switch it back to main branch.
> 
> (and I have now reverted it back to main)

@mig5 I reverted back to main before this was merged and a subsequent run didn't have this change included. Can you confirm that when PR is merged in this repo it still needs a manual `git pull`? That's what I did to hopefully fix it, but I'm wondering if that step is expected or some broken functionality somewhere.

I mean, it's not too critical and we can keep it manual until OpenQA replaces this.

mig5 commented:
@deeplow Indeed the webhook POST to the server from Github received a 400 Bad Request response. It is not clear why. I just re-entered the webhook secret in the interface and tried a redelivery and now it works. It's possible the webhook secret was out of date. Sorry about that. Should be ok from here.
